docs/cookbook/platform-sh.rst

```diff
@@ -242,7 +248,13 @@ When you get connected please run:
 
 .. code-block:: bash
 
-    $ php bin/console sylius:install
+    $ php bin/console sylius:install --env prod
+
+.. warning::
+
+    By default platform.sh creates only one instance of database with the `main` name. Installing Sylius on dev environment would required
+`main_dev` which of course will end up with an exception. Deploying development environment is possible but will require a little bit more
```

Comment:
also here indent is missing. All text that you want to be in the warning, has to be indented.